sys/net/routing/rpl/rpl.c

```diff
@@ -116,6 +110,11 @@ uint8_t rpl_init(int if_id, ipv6_addr_t *address)
     return SIXLOWERROR_SUCCESS;
 }
 
+kernel_pid_t rpl_get_if(void)
```

Comment:
I remember a conversation with  @OlegHahm citing @Kijewski that C is not Java (: Maybe expose rpl_if_id instead of using getter/setter?